Mage.Sets/src/mage/cards/f/FogOfWar.java

```diff
@@ -0,0 +1,45 @@
+package mage.cards.f;
+
+import mage.abilities.dynamicvalue.common.PermanentsOnBattlefieldCount;
+import mage.abilities.effects.common.GainLifeEffect;
+import mage.abilities.effects.common.PreventAllDamageByAllPermanentsEffect;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.ComparisonType;
+import mage.constants.Duration;
+import mage.filter.common.FilterCreaturePermanent;
+import mage.filter.predicate.mageobject.PowerPredicate;
+
+import java.util.UUID;
+
+/**
+ * @author PurpleCrowbar
+ */
+public final class FogOfWar extends CardImpl {
+
+    private static final FilterCreaturePermanent filter = new FilterCreaturePermanent("creatures with power 3 or less");
+    private static final FilterCreaturePermanent filter2 = new FilterCreaturePermanent("creatures on the battlefield");
```

Comment:
Same here.  Should be "for each creature" not "creatures" (for filter2.  other filter is fine).